scripts/automation/allocate-to-milestone-optimized.js

```
@@ -0,0 +1,552 @@
+    process.exit(result.success ? 0 : 1);
+  } catch (err) {
+    console.error(`Fatal error: ${err.message}`);
+    process.exit(1);
+  }
+}
+
+// Export for test use
+export { MilestoneAllocator, AllocationError };
+
+// CLI entry point
+if (typeof eval !== "undefined") {
```

Biome (2.5.8)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)